### DISCUSSION TITLE
Lock background scrolling while modal is open

### DISCUSSION THREAD

**Discussion by notscottsmith** [wire-elements/modal#211] [Q&A]
I have got several modals I'm using which take up more than the viewport height on mobile as well as a signature pad in some others - in both cases using the thumb to move the modal up and down can occasionally scroll the background and when trying to sign the signature pad, the background will also scroll (making it extremely difficult to sign the signature pad).

Is there a way to lock the background so that this doesn't happen? Or, are these two separate issues displaying as a similar problem?

**Comment by thoasty-dev** [comment]
Add the ``overflow-y-hidden`` class to the tailwind safelist.
